python/rmm/rmm.py

```diff
@@ -237,6 +237,19 @@ def rmm_cupy_allocator(nbytes):
     return ptr
 
 
+try:
+    from torch.cuda.memory import CUDAPluggableAllocator
+except ImportError:
+    rmm_torch_allocator = None
+else:
+    _alloc_free_lib_path = rmm._lib.torch_allocator.__file__
+    rmm_torch_allocator = CUDAPluggableAllocator(
+        _alloc_free_lib_path,
+        alloc_fn_name="allocate",
+        free_fn_name="deallocate",
+    )
```

Comment:
Q: Would this honor `rmm.reinitialize()` if a user changes the MR?

Reply:
`rmm.reinitialize()` resets the default memory resource used by RMM. Each call to `allocate()` and `deallocate()` queries the default memory resource via a call to `get_current_device_resource()`, so -- yes.